File: app/src/main/java/fr/insapp/insapp/fragments_intro/IntroClubsFragment.java
package fr.insapp.insapp.fragments_intro;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import fr.insapp.insapp.R;

/**
 * Created by thoma on 03/12/2016.
 */

public class IntroClubsFragment extends Fragment {

    public IntroClubsFragment() {
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {

        View rootView = inflater.inflate(R.layout.fragment_intro_association, container, false);
        return rootView;
    }
}

File: app/src/main/java/fr/insapp/insapp/fragments/intro/IntroClubsFragment.kt
package fr.insapp.insapp.fragments.intro

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import fr.insapp.insapp.R

/**
 * Created by thomas on 03/12/2016.
 */

class IntroClubsFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_intro_clubs, container, false)
    }
}
